scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -15,7 +15,6 @@
 #
```

Comment:
please, remove `scripts/install_dependencies/What`

Reply:
Already in this PR